kafka-avro-schema-quickstart/README.md

```diff
@@ -0,0 +1 @@
+The corresponding guide: https://github.com/quarkusio/quarkus/pull/16503
```

Comment:
Don't forget to update it to the final URL.

Reply:
oops :D